src/pages/contributors/alexander-brandes.adoc

```diff
@@ -0,0 +1,81 @@
+= Alexander Brandes
+:page-name: Alexander Brandes
+:page-linkedin: 
+:page-twitter: NotMyFault_OG
+:page-github: NotMyFault
+:page-email: 
+:page-image: avatar/alexander-brandes.jpg
+:page-pronouns: He/Him
+:page-location: Kassel, Germany
+:page-firstcommit: 2019
+:page-featured: true
+:page-intro: Alexander Brandes is an open-source contributor and maintainer and has serves on the Jenkins governance board. He has made contributions to multiple aspects of the Jenkins project including Jenkins core, multiple plugins, weekly and long term support releases, and the community.
+
+When he is not pursuing open-source endeavors, Alexander is a passionate musician and has played the trumpet for more than 15 years, primarily playing jazz and film music. In addition to being the lead trumpet in a band, he also teaches beginners how to play an instrument.
```

Comment:
```suggestion
When he is not pursuing open-source endeavors, Alexander is a passionate musician and is playing the trumpet since more than 15 years, primarily playing jazz and film music. In addition to being the lead trumpet in a band, he also teaches beginners how to play an instrument.
```
Present tense here too.

Reply:
"for more than" reads better than "since more than", no?

Reply:
```suggestion
When he is not pursuing open-source endeavors, Alexander is a passionate musician and has been playing the trumpet for more than 15 years, primarily playing jazz and film music. In addition to being the lead trumpet in a band, he also teaches beginners how to play an instrument.
```
So I updated the text a bit to be more in line with the verbiage, without changing the tense completely. In this case, the present tense is a bit awkward so I think this answers the suggestion from @lemeurherve while keeping the sentence grammatically correct.